src/metabase/integrations/ldap/default_implementation.clj

```diff
@@ -145,9 +145,9 @@
                        (assoc :is_active true)))]
     (u/prog1 new-user
       (when sync-groups?
-        (let [group-ids   (ldap-groups->mb-group-ids groups settings)
+        (let [group-ids            (ldap-groups->mb-group-ids groups settings)
               all-mapped-group-ids (all-mapped-group-ids settings)]
-          (integrations.common/sync-group-memberships! new-user group-ids all-mapped-group-ids false))))))
```

Comment:
here is the only part that is actually changed in this file, the rest are indentation.